CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
+
+### Release notes
+
+* Features
+  * Kubernetes 1.28 is GA.
+  * KEDA addon is GA.
+  * Added kubernetes patch versions 1.25.15, 1.26.10, 1.27.7a
+* Preview Features
+  * [dropgz](https://github.com/Azure/azure-container-networking/tree/master/dropgz) is enabled for Windows for the overlay subnet feature, in 2 regions: eastus2euap and centraluseuap
+  * Cluster network settings can be updated to enable Kubenet -> CNI Overlay migration - available in the [CLI](https://github.com/Azure/azure-cli-extensions/pull/6936)
+* Bug Fixes 
+  * Under some conditions it was possible to set `max_surge=0` which may interfere with upgrades.  Now `max_surge` must be > 0.  See [Customize node surge upgrade](https://learn.microsoft.com/en-us/azure/aks/operator-best-practices-run-at-scale#cluster-upgrade-considerations-and-best-practices) for more information about the setting.
+  * In some occasions, PUT operations on managedClusters or agentPools see long latency in the operations.  This fixes one of a few scenarios.
+  * PATCH operations were allowed on managedClusters in a non-terminal provisioningState.  This could cause an eTag mismatch and inconsistent results or failures.  PATCH operations will now be block for managedClusters in a non-terminal provisioningState.
+* Behavioral Change
+  * Updates to optimize the kube-reserved eviction thresholds available in 1.28
+* Component Updates
+  * Update the aks-app-routing-operator to [version 0.0.7](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#007---2023-11-04) which includes notable changes in [version 0.0.6](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#006---2023-10-27).
+    * This update has 3 CVE fixes for the nginx ingress controller.
+      * [CVE-2023-5044](https://nvd.nist.gov/vuln/detail/CVE-2023-5044)
+      * [CVE-2022-4886](https://nvd.nist.gov/vuln/detail/CVE-2023-4886)
+      * [CVE-2023-5043](https://nvd.nist.gov/vuln/detail/CVE-2023-5043)
+    * The following changes are also included:
+      * The AJP protocol is no longer supported.
+      * The `whitelist-source-range` annotation has been renamed to `allowlist-source-range`. Both are currently supported but it is recommended to move to the new annotation `allowlist-source-range`.
+    * The `custom-http-errors` annotation now only supports errors between 400 and 599.
+  * Azure Monitor Metrics [November release](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-11-03-2023)
+  * Update [gatekeeper to v3.13.3](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.13.3) and policy addon 1.2.1
+    * Azure Policy Changes
+      * Introduce warn for policies, available in select upcoming built-in policy experiences
+      * Show an exempt ComplianceReasonCode in the portal for exempt policies.
+  * Update Azure Disk CSI driver version to v1.29.1 on AKS 1.28, to v1.28.4 on AKS 1.27, to v1.26.7 on AKS 1.26 and 1.25
```

Comment:
Is there any documentation about these version bumps?

Reply:
added links to relnotes